appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -81,32 +88,50 @@ for ($i = 0; $i -lt $sarifLog.runs.Count; $i++){
     #Codeflow
     for($k = 0; $k -lt $sarifLog.runs[$i].results[$j].codeFlows.Count; $k++){
       for($l = 0; $l -lt $sarifLog.runs[$i].results[$j].codeFlows[$k].locations.Count; $l++){
-        $sarifLog.runs[$i].results[$j].codeFlows[$k].locations[$l].physicalLocation.uri = Rebase-Uri $sarifLog.runs[$i].results[$j].codeFlows[$k].locations[$l].physicalLocation.uri
+        $uri = $sarifLog.runs[$i].results[$j].codeFlows[$k].locations[$l].physicalLocation.uri
+        if($uri){
+          $sarifLog.runs[$i].results[$j].codeFlows[$k].locations[$l].physicalLocation.uri = Rebase-Uri $uri
+        }
       }
     }
 
     for($k = 0; $k -lt $sarifLog.runs[$i].results[$j].relatedLocations.Count; $k++){
-      $sarifLog.runs[$i].results[$j].relatedLocations[$k].physicalLocation.uri = Rebase-Uri $sarifLog.runs[$i].results[$j].relatedLocations[$k].physicalLocation.uri
+      $uri = $sarifLog.runs[$i].results[$j].relatedLocations[$k].physicalLocation.uri
+      if($uri){
+        $sarifLog.runs[$i].results[$j].relatedLocations[$k].physicalLocation.uri = Rebase-Uri $uri
+      }
     }
 
     for($k = 0; $k -lt $sarifLog.runs[$i].results[$j].stacks.Count; $k++){
-      $sarifLog.runs[$i].results[$j].stacks[$k].frames.uri = Rebase-Uri $sarifLog.runs[$i].results[$j].stacks[$k].frames.uri
+      $uri = $sarifLog.runs[$i].results[$j].stacks[$k].frames.uri
+      if($uri){
+        $sarifLog.runs[$i].results[$j].stacks[$k].frames.uri = Rebase-Uri $uri
+      }
     }
 
     for($k = 0; $k -lt $sarifLog.runs[$i].results[$j].fixes.Count; $k++){
-      $sarifLog.runs[$i].results[$j].fixes[$k].fileChanges.uri = Rebase-Uri $sarifLog.runs[$i].results[$j].fixes[$k].fileChanges.uri
+      $uri = $sarifLog.runs[$i].results[$j].fixes[$k].fileChanges.uri
+      if($uri){
+      $sarifLog.runs[$i].results[$j].fixes[$k].fileChanges.uri = Rebase-Uri $uri
```

Comment:
Sorry, no, I meant "maybe there was a tab there, and that's why the indentation was wrong." Please use spaces :-)

---
In reply to: [71192075](https://github.com/Microsoft/sarif-sdk/pull/445#discussion_r71192075) [](ancestors = 71192075)